ch05/01_main-chapter-code/ch05.ipynb

```diff
@@ -260,7 +260,7 @@
     "id": "0f3d7ea2-637f-4490-bc76-e361fc81ae98"
    },
    "source": [
-    "### 5.1.2 Calculating the text generation loss: cross entropy, and perplexity"
+    "### 5.1.2 Calculating the text generation loss: cross-entropy and perplexity"
```

Comment:
It's funny, the publisher changed all instances "cross entropy" to "cross-entropy" in the layouted book 😅. I do prefer "cross entropy" in the Jupyter notebooks though, so thanks for the change!

PS: All instances of finetuning were also changed to fine-tuning ... 🥲

Reply:
Yeah, nothing ground-breaking, therefore if have labeled it with "minor". The crucial point here was to fix the variable/function names. Don't worry, no more fixes to come 😄